resources/assets/components/CampaignIdSingle/index.js

```diff
@@ -81,8 +82,7 @@ class CampaignIdSingle extends React.Component {
             <p>–</p>
           )}
           <h4>Start Date</h4>
-          <p>{new Date(campaign.start_date).toDateString()}</p>
-
+          <p>{parse(campaign.start_date)}</p>
```

Comment:
@DFurnes I think this is the best way to do this since `campaign.start_date` is a date but when I do this, I get the error: 

```
index.js:319 Uncaught TypeError: 3 arguments required, but only 1 present
    at parse (index.js:319)
```

According to the [documentation](https://date-fns.org/v1.9.0/docs/parse), I think it only takes one argument?

Reply:
oh wait nvm I see now that I was on the wrong version of the documentation!